pages/issuer/issue.vue

```diff
@@ -39,7 +39,9 @@
     <div class="w-72 bg-gray-100 p-6 bg-gray-300 -mt-8">
       <h2 class="text-xl font-semibold">Updates</h2>
       <ol class="list-decimal">
-        <li v-for="message in logMessages">{{ message }}</li>
+        <li v-for="message in logMessages">
+          [{{ message.source }}<p v-if="message.target"> -> {{ message.target }}</p>] {{ message.message }}
```

Comment:
Just in case you have a target called `'false'`, then I think it would not display...

```suggestion
          [{{ message.source }}<p v-if="message.target === undefined"> -> {{ message.target }}</p>] {{ message.message }}
```